Develop

AlexanderSchober opened the pull request.

gpospelov commented:
My suggestion for the future is to switch in directory and file names all to lower case, to follow PEP8 recommendation

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

As for the rest - congratulations with first pull request, great job!

AlexanderSchober committed:
Support strings as parameter keys

AlexanderSchober committed:
Removed markers

AlexanderSchober committed:
Added the button to add and remove environments

AlexanderSchober committed:
Added the posiibility to remove curves without deleting them

AlexanderSchober committed:
More coherence with the environments

AlexanderSchober committed:
Selected environment shows in the window title

AlexanderSchober committed:
Validate will now update the scipt page

AlexanderSchober committed:
Fixed bug where the load checkboxes would not trigger the NodeLoad

AlexanderSchober committed:
changed the default state

AlexanderSchober committed:
Fixed bug where io menus became inaccessible

AlexanderSchober committed:
The script page will no initially show the first tab

AlexanderSchober committed:
The phase mask selector was left out of updates: Fixed

AlexanderSchober committed:
Selector in panel will update with external mask change

AlexanderSchober committed:
The panel is now updated according to the process stage

AlexanderSchober committed:
Fixed truncated echo times by switching to scientific formating

AlexanderSchober committed:
Enhanced GUI view and usability in the script part

AlexanderSchober committed:
Fixing the example notebook to the new additions

AlexanderSchober committed:
New version 0.1.0

AlexanderSchober committed:
Fixed test to account for version change

AlexanderSchober committed:
Fixed install recquirements

AlexanderSchober committed:
Reworked the scripts and fixed a ploting visual

AlexanderSchober committed:
Removed depreciated files

Commit:
Merge pull request #19 from AlexanderSchober/develop

Release 0.1.0

AlexanderSchober committed:
Merge pull request #20 from scgmlz/develop

Release 0.1.0

AlexanderSchober committed:
Supposedly changed self closing dialogs under Windows

AlexanderSchober committed:
Fixed the dialog window creation to work on Windows

AlexanderSchober committed:
Cleaned up the meta_widget class

AlexanderSchober committed:
Added the ability to manually overwrite the metadata

AlexanderSchober committed:
Removed the shapely requirement on instllation

AlexanderSchober committed:
Fixed refresh behaviour in masks

AlexanderSchober committed:
Fixed Preview hiding behaviour

AlexanderSchober committed:
Consolidated the panel behaviours

AlexanderSchober committed:
Fixed potential bug with default meta data setup and saving

AlexanderSchober committed:
Fixed script related saving

AlexanderSchober committed:
Added no environment dialog

AlexanderSchober committed:
Added the ability to add folder to search data in other locations

AlexanderSchober committed:
Code cleanup

Trying to reach a general clean state of the codebase by homogenising the comments and removing extra lines.

AlexanderSchober committed:
Windows bug fix when the path was ''

AlexanderSchober committed:
Added small dialog

AlexanderSchober committed:
Moved mask definition to the phase script

AlexanderSchober committed:
Fixed mask overwriting bug

AlexanderSchober committed:
Changed plot precision on request

AlexanderSchober committed:
Fixed a bug involving the mask addition in windows

AlexanderSchober committed:
Fixed color issues in the environment tab

AlexanderSchober committed:
Fixed cosmetic issues in the mask selection

AlexanderSchober committed:
Cosmetic changes to data import

AlexanderSchober committed:
Fixed bug where the absence of an environment would not allow to load

AlexanderSchober committed:
Visual changes

AlexanderSchober committed:
Fixed path management in windows

AlexanderSchober committed:
Visual modifications

AlexanderSchober committed:
Fixed color problems with the main window background

AlexanderSchober committed:
Fixed bug where import would not load data

AlexanderSchober committed:
Edited gitignore to be more futureproff

AlexanderSchober committed:
Cosmetic changes

AlexanderSchober committed:
Global result visualizer installed into the system

AlexanderSchober committed:
Removed leftover result parts in the script machinery

AlexanderSchober committed:
Visual updates and added an icon

AlexanderSchober committed:
Corrected Windows bug for color selector

AlexanderSchober committed:
Fixed extra folder path bug under windows

AlexanderSchober committed:
Windows related fixes

AlexanderSchober committed:
Redid the mask visual system based on qtmodels

- The model can now be set to the treeview in place and then propagates through the GUI
- The items are all editable through the widget interface
- The mask list automatically updates in the script components

AlexanderSchober committed:
Removed obsolete code

AlexanderSchober committed:
Fixed a bug of mask management in the scrips

AlexanderSchober committed:
Added default save path to the postscript module

AlexanderSchober committed:
Ability to investigate contrast without correcting the phase

AlexanderSchober committed:
Removed dead code

AlexanderSchober committed:
Introduced the code editors with line numbers

AlexanderSchober committed:
Fixed mask tests to cpmly with new system

AlexanderSchober committed:
Upgraded to version 0.1.1

AlexanderSchober committed:
Fixed overriden mask state in the script section

AlexanderSchober committed:
Minor bug fixes in the script handling

AlexanderSchober committed:
Added feature of All information used to compute the echo

AlexanderSchober committed:
Visual fix of the color of the logo label

Commit:
Merge pull request #21 from AlexanderSchober/develop

Release 0.1.1

AlexanderSchober committed:
Merge pull request #22 from scgmlz/develop

Release 0.1.1

AlexanderSchober committed:
Resources are now auto-generated on launch if not found


Resources are now auto-generated on launch if not found

AlexanderSchober committed:
Introduced test launch

AlexanderSchober committed:
Test apveyor

AlexanderSchober committed:
test Appveyor

AlexanderSchober committed:
Test Apveyor

AlexanderSchober committed:
test Apveyor

AlexanderSchober committed:
Test Apveyor

AlexanderSchober committed:
Test Appveyor

AlexanderSchober committed:
TestAppvyor

AlexanderSchober committed:
Test Appveyor

AlexanderSchober committed:
Test Appveyor

AlexanderSchober committed:
test

AlexanderSchober committed:
test

AlexanderSchober committed:
test

AlexanderSchober committed:
test

AlexanderSchober committed:
test

AlexanderSchober committed:
Setup of Appveyor

AlexanderSchober committed:
Setup of Appveyor

AlexanderSchober committed:
Fixing travis

AlexanderSchober committed:
Fixing travis

AlexanderSchober committed:
Fixing Travis

AlexanderSchober committed:
Fixing Travis

AlexanderSchober committed:
Travis

AlexanderSchober committed:
travis

AlexanderSchober committed:
Travis

AlexanderSchober committed:
test travis

AlexanderSchober committed:
travis

AlexanderSchober committed:
travis

AlexanderSchober committed:
parse

AlexanderSchober committed:
teest

AlexanderSchober committed:
lol

AlexanderSchober committed:
yo

AlexanderSchober committed:
travis

AlexanderSchober committed:
ubuntu

AlexanderSchober committed:
test

AlexanderSchober committed:
change

AlexanderSchober committed:
test

AlexanderSchober committed:
lol

AlexanderSchober committed:
trial

AlexanderSchober committed:
try

AlexanderSchober committed:
try

AlexanderSchober committed:
Set all the unittesting for windows

AlexanderSchober committed:
Merge remote-tracking branch 'origin/develop' into develop

AlexanderSchober committed:
Minor feature request

AlexanderSchober committed:
Modification to function names, descriptions and tests

AlexanderSchober committed:
Refactoring of the phase correction

Reconstruction of the functions as well as including a function library. Multiprocessing was also put in place.

AlexanderSchober committed:
Refactoring the phase correction

AlexanderSchober committed:
Merge remote-tracking branch 'origin/Rewrite_core' into Rewrite_core

AlexanderSchober committed:
Finished rewriting of the phase processing

AlexanderSchober committed:
FInishing rewriting the core

AlexanderSchober committed:
Rewriting of the core functionalities finished

AlexanderSchober committed:
Edited gitignore to allow for text files

AlexanderSchober committed:
edited test file and travis

AlexanderSchober committed:
Introduced the multiprocessing argument into fit parameters

AlexanderSchober committed:
Removed multiprocessing from Travis

Added the possibility to run the code in single processor mode which would not spawn any multiprocessing entities

AlexanderSchober committed:
Re-linked GUI with the new core

AlexanderSchober committed:
formatting only

AlexanderSchober committed:
instrument structure now added


Instrument Structure added

AlexanderSchober committed:
Restructured layout of the Script widget

AlexanderSchober committed:
Linked the GUI with the refactored core

This includes the inclusion of some new features such as the high exposure correction. Furthermore the GUI has been slightly corrected to include the instrument selection.

AlexanderSchober committed:
Fixed the high exposure routines

AlexanderSchober committed:
Updated default elements for processing

AlexanderSchober committed:
Fixed the Panel and extracted the Panel worker into a different file

AlexanderSchober committed:
Fixed Panel functionalities

AlexanderSchober committed:
Reorganized the core script manager

AlexanderSchober committed:
Fixed script implementation

AlexanderSchober committed:
Moved some script processing components from the GUI to the core

AlexanderSchober committed:
Added removal of trailing lines to core scripts

AlexanderSchober committed:
Re-organised GUI folder and fixed imports (ignore review)

AlexanderSchober committed:
Merge branch 'Rewrite_core' into develop

AlexanderSchober committed:
Re-implemented the last version of SimplePlot

AlexanderSchober committed:
Rewrote the mask management

AlexanderSchober committed:
Fixed tests to run with the new mask generator

AlexanderSchober committed:
Improved simpleplot

AlexanderSchober committed:
Fixed SimplePlot integration

AlexanderSchober committed:
Log connectivity

AlexanderSchober committed:
Fixed slow cursor (temp solution)

AlexanderSchober committed:
Added the option to turn off live computation

This feature aims at not locking the interface every time an uodate is performed in the panel

AlexanderSchober committed:
Fixed threading memory leak

AlexanderSchober committed:
Removed line highlite

AlexanderSchober committed:
Small modification to text colors

AlexanderSchober committed:
Fixed use of deprecated imread

AlexanderSchober committed:
Performance improvements and new deatures

AlexanderSchober committed:
Merge branch 'develop' of https://github.com/AlexanderSchober/NSE_Soft into develop

AlexanderSchober committed:
Plot visual adaptations

AlexanderSchober committed:
Now version 0.2.0

AlexanderSchober committed:
Cosmetics

AlexanderSchober committed:
Changed the fit axis to more usable in log mode

AlexanderSchober committed:
Fixed names of result plots not being diplayed

AlexanderSchober committed:
Using a button in an environment will now select it

AlexanderSchober committed:
Temporary fix of foil mamagement

AlexanderSchober committed:
MIEZEPY now checks for sanity before validating

AlexanderSchober committed:
Introduced event filters to manage focus of QListWidgetItems

AlexanderSchober committed:
Implementation of Drag and drop of data masks and scrips

AlexanderSchober committed:
Cleaning of various files

AlexanderSchober committed:
Added foil select to the script panel

AlexanderSchober committed:
Added script reset buttons to reset wrong scripts

AlexanderSchober committed:
Removed the foil to select dialog as it is redundant

AlexanderSchober committed:
Fixed tristate check-boxes not behaving as expected for foil select

AlexanderSchober committed:
Switched the foil select system to a model based approach

AlexanderSchober committed:
Fixed drag and drop and some cosmetic

AlexanderSchober committed:
Complete remake of the result analysis section

AlexanderSchober committed:
Finished new Result visualizer

AlexanderSchober committed:
Fixed version test

AlexanderSchober committed:
Fixed the Tests

AlexanderSchober committed:
Added 4D visualisation

Commit:
Merge pull request #24 from AlexanderSchober/develop

Version 0.2.0

AlexanderSchober committed:
Merge pull request #25 from scgmlz/develop

Version 0.2.0

AlexanderSchober committed:
Fixed Issue with absent __init__.py

The absence of this files made the installation of the miezepy package through 'python setup.py install' incomplete. This has been resolved.

AlexanderSchober committed:
Minor changes for 4D visualisation

AlexanderSchober committed:
Fixed log with 0 limit

AlexanderSchober committed:
Introduced the detector foil visualisation

AlexanderSchober committed:
Added the scale to all 2 and 3d plot

AlexanderSchober committed:
Fixed the Pointer implementation

AlexanderSchober committed:
moved to version 0.2.1

AlexanderSchober committed:
Fixed the version test

Commit:
Merge pull request #26 from AlexanderSchober/develop

version 0.2.1

AlexanderSchober committed:
Merge pull request #27 from scgmlz/develop

Version 0.2.1

AlexanderSchober committed:
Adapted miezepy to the next version of simpleplot

AlexanderSchober committed:
Fixed some foil numbering issues

AlexanderSchober committed:
transposed the detector image as it was probably displayed with the wrong major

AlexanderSchober committed:
Some esthetic in the panel

AlexanderSchober committed:
Shoudl have fixed the gemoetry issue

AlexanderSchober committed:
Fixed compatibility with new version of simpleplot

AlexanderSchober committed:
Changing the way masks are handled

AlexanderSchober committed:
Made some adjustements for the safety of the mask parameter items

AlexanderSchober committed:
Added the rectangle shape processing

AlexanderSchober committed:
Added the pie mask generator

AlexanderSchober committed:
Added the triangle shape

AlexanderSchober committed:
Deleted obsolete file

AlexanderSchober committed:
Added the ellipse mask item

AlexanderSchober committed:
Fixed mask model generator

AlexanderSchober committed:
Fixed the ability to remove mask elements

AlexanderSchober committed:
Modified the new mask system

AlexanderSchober committed:
Small cleanup

AlexanderSchober committed:
Introduced the data view into the mask designer

AlexanderSchober committed:
Finished modifications on the panel

AlexanderSchober committed:
The Echo time dropdown has been made beatiful

AlexanderSchober committed:
Fixed multiple high exposure files

AlexanderSchober committed:
Reworked the pointer on the canvas

AlexanderSchober committed:
Fixed the panel

AlexanderSchober committed:
Updated to version 0.3

AlexanderSchober committed:
Merge branch 'develop' into develop

AlexanderSchober committed:
Cleanup

AlexanderSchober committed:
Merge branch 'develop' of https://github.com/AlexanderSchober/NSE_Soft into develop

AlexanderSchober committed:
Attempt at fixing Travis CI

AlexanderSchober committed:
Merge pull request #29 from AlexanderSchober/develop

Version 0.3.0

AlexanderSchober committed:
Fixed the high exposure foil not showing

AlexanderSchober committed:
Raw data now also updates with key events

AlexanderSchober committed:
Fixed the raw data list not clearing whe entering a new environment

AlexanderSchober committed:
Fixed the unability to remove some meta elements in the raw data loader

AlexanderSchober committed:
Updated to version 0.3.1

AlexanderSchober committed:
Merge pull request #31 from AlexanderSchober/develop

Version 0.3.1

AlexanderSchober committed:
Merge pull request #30 from scgmlz/develop

Version 0.3.1

Commit:
Update the tests and code for the new iminuit 2.5 library

Commit:
Cleanup of the file fit_mieze_contrast

Commit:
Add all the tests for the combineData method

Commit:
Add Test for the fitContrastSinus method

Commit:
Add Test for the fitContrastSinus method

Commit:
Add Test for the fitContrastSinus method

Commit:
Set the contrast correction to use individual foild with weights

Commit:
Small fixes to data types

Commit:
Major improvements to installation

AlexanderSchober committed:
Merge branch 'master' into develop

Commit:
Merge master into develop

Commit:
Merge branch 'develop' of https://github.com/AlexanderSchober/NSE_Soft into develop

Commit:
Small inegration fix

AlexanderSchober committed:
Update miezepy_tests.yml

AlexanderSchober committed:
Update miezepy_tests.yml

AlexanderSchober committed:
Merge branch 'master' into develop

AlexanderSchober committed:
Update miezepy_tests.yml

AlexanderSchober committed:
Update miezepy_tests.yml

AlexanderSchober committed:
Update miezepy_tests.yml

AlexanderSchober committed:
Update miezepy_tests.yml

Commit:
tests
